### ISSUE TITLE
 Checkboxes on Series tab are not labelled

### ISSUE THREAD

**Issue by luniki** [opencast/opencast-admin-interface#595]
Form elements must have labels. The checkboxes on the Series tab (one per row and the check-all checkbox at the top) do not have them. This could be an implicit or explicit <label>, an aria-label or title attribute.

This has critical impact on blind/low vision people.
